src/numerics/Func1.cpp

```diff
@@ -223,6 +223,39 @@ Poly1::Poly1(const vector<double>& params)
     copy(params.data(), params.data() + m_cpoly.size(), m_cpoly.begin());
 }
 
+string Poly1::write(const string& arg) const
+{
+    // write terms in reverse order
+    string out = "";
+    if (m_cpoly[m_cpoly.size()-1] != 0.) {
+        out = fmt::format("{}", m_cpoly[m_cpoly.size()-1]);
```

Comment:
Do these lines need the `fmt::runtime` changes that Ray added in #1739?

Reply:
I don't believe so - if I understand correctly, `fmt::runtime` is only required when there are non-constant formatting strings involved?